### PR TITLE
Build the homepage content with suru strips

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Updated some base template content like adding favicons
Built the base page with content from the design
Wrote a number of suru strip styles

## QA
- Go to http://0.0.0.0:8002/
- Check it matches [the design](https://drive.google.com/file/d/114QpYLu8SXvqd_5f8caFOvsrC0TTuSSW/view)

## Screenshot
Screenshot_2019-07-18 Canonical The company behind Ubuntu

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1371

### CODE REVIEW

**Comment by barrymcgee** [comment]
@anthonydillon You now have conflicts here after the footer branch landed.

**Comment by barrymcgee** [comment]
This looks off:

Screenshot 2019-07-18 at 09 58 12

**Comment by anthonydillon** [comment]
@barrymcgee thanks, fixed you your points and converted the slants to clipping paths.

File: templates/index.html [hunk 1]
```diff
@@ -1,7 +1,309 @@
...
+          <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/02c32f74-snapcraft-logo.png" alt="Snapcraft logo">
+          <hr class="u-sv1">
+          <div class="p-card__content">
+            <p>The app store with secure packages and ultra-reliable updates for multiple Linus distros</p>
+            <p><a href="https://snapcraft.io">snapcraft.io&nbsp;&rsaquo;</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row u-equal-height  u-extra-space">
```
Comment (barrymcgee) [inline templates/index.html]: Ironically, there is an extra space here before `u-extra-space`

```suggestion
    <div class="row u-equal-height u-extra-space">
```

**Comment by barrymcgee** [comment]
These links are not accessible nor do they match the design - you might want to tackle here or separately, up to you.

Screenshot 2019-07-19 at 12 20 14

**Comment by Lukewh** [comment]
Still looking for a design review @anasereijo @waynecrosby

**Comment by waynecrosby** [comment]
Looking good, a few things to fix up:
1) Update header graphic with https://assets.ubuntu.com/v1/670abb00-header_illustration.svg
2) Top right suru strip should be #4A183C @30% opacity
3) Products section, I know you've used Vanilla but can we match the design using bespoke css
4) Products section update: Mark wanted to break Security & Support out so there is a new design (see below)
5) Careers section: add the line under the title, match the size of quotes and make white, quote copy should be 20px, use an outlined white button

**Comment by waynecrosby** [comment]
1) Reduce line-height space in header, bring 'open source, everywhere' up a bit if possible.
2) Add fullstops after 'everywhere'.
3) Add fullstops after 'applications'. 
4) Please replace the logos in the Products list with the ones below, my fault:
https://assets.ubuntu.com/v1/9e2b2490-products-kubernetes-wht-aubergine2.svg
https://assets.ubuntu.com/v1/9d654941-products-lxd-wht-aubergine2.svg
https://assets.ubuntu.com/v1/9b099dad-products-openstack-wht-aubergine2.svg
https://assets.ubuntu.com/v1/81c6d8e6-products-security-wht-aubergine3.svg
https://assets.ubuntu.com/v1/7cd18f89-products-snapcraft-wht-aubergine2.svg
https://assets.ubuntu.com/v1/5513fa39-products-support-wht-aubergine2.svg
5) Add fullstops after 'open source' in careers section. 
6) Add Youtube logo to social icons in the footer.

File: static/sass/_pattern_headings.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      margin-bottom: 1rem;
+    }
+
+    &::after {
+      background-color: $color-x-light; 
+      content: '';
+      display: block;
+      height: 1px;
+      width: 75px;
+      margin-bottom: 1rem;
```
Comment (sowasred2012) [inline static/sass/_pattern_headings.scss]: CSS properties should be sorted alphabetically (`margin-bottom` before `width`)

File: static/sass/_pattern_strip.scss [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      100%;
+    color: $color-x-light;
+    position: relative;
+  }
+}
+
+@mixin canonical-p-strip-suru {
+  .p-strip--suru {
+    @extend %vf-strip;
+    padding: 8rem 0;
```
Comment (sowasred2012) [inline static/sass/_pattern_strip.scss]: CSS properties should be sorted alphabetically (`padding` after `color`)

File: static/sass/styles.scss [hunk 4]
```diff
@@ -1,7 +1,33 @@
 @import 'vanilla-framework/scss/build';
 
+$color-brand: #7C355D;
+
 @import 'pattern_icons';
+@import 'pattern_strip';
```
Comment (sowasred2012) [inline static/sass/styles.scss]: I think the convention in other projects has been to alphabetise these `@include` and `@import` statements

File: static/sass/_pattern_cards.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      padding: calc(#{$spv-inner--large} - 1px);
+    }
+
+    .p-strip {
+      padding: calc(#{$spv-inner--large} - 1px);
+      background-color: $color-brand;
+      color: $color-x-light;
+    }
+  }
+}
```
Comment (sowasred2012) [inline static/sass/_pattern_cards.scss]: SCSS files should have an empty line at the end

File: static/sass/_pattern_headings.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    }
+  }
+}
```
Comment (sowasred2012) [inline static/sass/_pattern_headings.scss]: SCSS should have an empty line at the end

File: static/sass/_pattern_headings.scss [hunk 7]
```diff
@@ -0,0 +1,30 @@
+@mixin canonical-p-headings {
+  .p-heading-highlight--left {
+
```
Comment (sowasred2012) [inline static/sass/_pattern_headings.scss]: Can remove this empty line

**Comment by Lukewh** [comment]
Is it just me, or do some of the logos that are in circles appear cropped at the bottom?
Screenshot_2019-08-01 Canonical The company behind Ubuntu
They seem to get progressively flatter on the bottom from left to right.

**Pull request merged**